flowey/flowey_core/src/node.rs

```diff
@@ -2275,7 +2275,7 @@ pub mod private {
     pub static FLOW_NODES: [FlowNodeMeta] = [..];
 
     // UNSAFETY: linkme uses manual link sections, which are unsafe.
-    #[allow(unsafe_code)]
+    #[expect(unsafe_code)]
```

Comment:
shouldn't we be pushing the `// UNSAFETY` justifications into the `expect` lint itself? i.e: as a lint reason?

or have lint reasons not stabilized yet?

Reply:
i think they have? But either way that should be a separate PR, since it'll also require some xtask fmt work.

Reply:
fair enough

Reply:
I think actually the plan here would be to turn on clippy::allow_attributes_without_reason. Then we can delete our lint entirely.

Reply:
(it covers expect too despite the name)

Reply:
Are we tracking that north-star anywhere (either on our internal board, or publicly?)

Reply:
https://github.com/microsoft/openvmm/issues/300